### PR TITLE
Remove Kogito profile to productize required components

### PR DESCRIPTION
**Pull request by almope**
Signed-off-by: Alberto Morales Pérez <almorale@redhat.com>

**Thank you for submitting this pull request**

**JIRA**: (https://issues.redhat.com/browse/RHPAM-3393)

**Referenced Pull Requests**:
* https://github.com/kiegroup/drools-wb/pull/1454

**Business Central**: n/a

**VS Code**: n/a

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* Retest PR: <b>jenkins retest this</b>
* A full downstream build: <b>jenkins do fdb</b>
* A compile downstream build: <b>jenkins do cdb</b>
* A full production downstream build: <b>jenkins do product fdb</b>
* An upstream build: <b>jenkins do upstream</b>
</details>

### CODE REVIEW

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/pom.xml [hunk 1]
```diff
@@ -40,20 +40,7 @@
...
     <module>kie-wb-common-stunner-bpmn-project-showcase</module>
+    <module>kie-wb-common-stunner-bpmn-kogito-runtime</module>
   </modules>
 
-  <profiles>
-    <profile>
-      <id>kogito</id>
-      <activation>
-        <property>
-          <name>!productized</name>
```
Comment (Ginxo) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/pom.xml:50]: @almope 
Why don't we use `kogitoProductized` instead?

**Review by Ginxo — APPROVED** [review]
Check my proposal, but in general terms LGTM

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/pom.xml (hunk 1 above)
Comment (mareknovotny) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/pom.xml:50]: we need to build it in community too @Ginxo

Comment (almope) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/pom.xml:50]: @Ginxo adding a new property implies that for both community and product build we have to add that as a parameter. Previously the `productized` property avoids the plugin executions on productization builds while now we need in all kind of builds.

**Review by AntStephenson — APPROVED** [review]
LGTM, removing the profile entirely is a better solution since we no longer need to exclude this from product builds.

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**